<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>321</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string> Resource Import</string>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Scroll Text</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_video">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>   video</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_picture">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> picture</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_selectvideo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Video</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton_selectpicture">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Picture</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>251</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string> Parameter Configure</string>
   </property>
   <widget class="QCheckBox" name="checkBox_hideTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>168</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hidden Time and Date Area</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_hideTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hidden Title Area</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_hideScroText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>146</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hidden ScrollText Area</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_fullscreen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>122</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Full Screen</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_volume">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_bright">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_volume">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_bright">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_1_bright">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_1_bright">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2_bright">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2_bright">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_1_time">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>6000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2_time">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>6000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> volume</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string> bright </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>brigt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>bright</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>171</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string> Make Tools</string>
   </property>
   <widget class="QComboBox" name="comboBox_USB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_makeudisk">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Make UDisk</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Reset">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_copy">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy File</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
